_posts/2020-11-15-digital-ocean-first-impressions.md

```diff
@@ -0,0 +1,79 @@
+---
+title: "First impressions: Digital Ocean App Platform"
+date: 2020-11-15 00:00:00 +0800
+author: eric
+---
+
+When I [first learned](https://www.digitalocean.com/blog/introducing-digitalocean-app-platform-reimagining-paas-to-make-it-simpler-for-you-to-build-deploy-and-scale-apps/) about the new Digital Ocean App Platform, I was pretty excited to see Heroku have some competition! We use Heroku on many projects and love its simplicity, but hate how quickly it can get expensive.
+
+I attempted to set up one of our client's projects on this new platform to give it a spin. Here is my experience.
+
+
+<!--more-->
+
+
+# Digital Ocean vs. Heroku
+
+<div style="text-align: center;">
+    <img src="/assets/images/articles/digital-ocean.png">
+</div>
+
+| Pros | Cons |
+|------|------|
+| Compared to Heroku, there's quite a bit less to learn     | No free database to test against, must pay $7/mo    |
+| Scaling up a bit should be cheaper than Heroku     | Very few add ons     |
+| Fresh, has the opportunity to grow with community input     | May not support many non-trivial apps because of lacking add-ons     |
```

Comment:
```suggestion
| Fresh, has the opportunity to grow with community input     | May not support many 'complex' or 'enterprise' apps because of lacking add-ons     |
```
Non-trivial isn't very clear here